reolinkapi/handlers/api_handler.py

```diff
@@ -64,28 +68,47 @@ def login(self) -> bool:
         :return: bool
         """
         try:
+            # Suppress only the single warning from urllib3 needed.
+            requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
```

Comment:
maybe make this optional?

or what about just leaving this to the client to set this as an env variable

```shell
PYTHONWARNINGS="ignore:Unverified HTTPS request"
```